src/archivematicaCommon/debian/control

```diff
@@ -8,6 +8,6 @@ Homepage: http://archivematica.org
 
 Package: archivematica-common
 Architecture: i386 amd64 
-Depends: ${shlibs:Depends}, ${misc:Depends}, python, python-pip, python2.7-elementtree, python-mimeparse, python-dateutil
+Depends: ${shlibs:Depends}, ${misc:Depends}, python, python-pip, python2.7-elementtree, python-dateutil
```

Comment:
Do we still need `python2.7-elementtree, python-dateutil`? Are they still used and/or can they be installed from pip?